qutip/tests/test_gates.py

```diff
@@ -70,6 +73,71 @@ def testSwapGate(self):
         psi_res = swap() * swap() * psi_in
         assert_((psi_in - psi_res).norm() < 1e-12)
 
+    def test1QubitGates(self):
```

Comment:
The actual name of the test function is typically the most accessible "documentation" of the test in the log, so more descriptive is always better (any surrounding class will also appear).  It's not like you have to call them manually, so they really can be like 60 characters long if it helps.

I know that you've generally followed what's in the rest of the file, though - this is just a general comment.

Reply:
Fixed on ad74aeb